src/autoComplete/autoCompleteList.tsx

```diff
@@ -61,11 +63,10 @@ extends Component <AutoCompleteListProps, AutoCompleteListState> {
   onKeyboardEventArrowDown = (e:Event) => {
     e.preventDefault()
 
-    const itemsLength = this.props.items.length
-    if (!itemsLength) return
+    if (isEmpty(this.props.items)) return
```

Comment:
nit: avoid oneline if

Reply:
Another PR :) Here I just replaced elements, and I struggle to merge this PR because it's reinitialising the approvals all the time :D